package.json

```diff
@@ -41,10 +41,10 @@
     "publish-stable": "lerna version --force-publish",
     "lint-staged": "lint-staged",
     "next-with-deps": "./scripts/next-with-deps.sh",
-    "next": "node --trace-deprecation --enable-source-maps packages/next/dist/bin/next",
-    "next-no-sourcemaps": "node --trace-deprecation packages/next/dist/bin/next",
-    "clean-trace-jaeger": "rm -rf test/integration/basic/.next && TRACE_TARGET=JAEGER node --trace-deprecation --enable-source-maps packages/next/dist/bin/next build test/integration/basic",
-    "debug": "node --inspect packages/next/dist/bin/next",
+    "next": "cross-env NEXT_TELEMETRY_DISABLED=1 node --trace-deprecation --enable-source-maps packages/next/dist/bin/next",
+    "next-no-sourcemaps": "cross-env NEXT_TELEMETRY_DISABLED=1 node --trace-deprecation packages/next/dist/bin/next",
+    "clean-trace-jaeger": "rm -rf test/integration/basic/.next && TRACE_TARGET=JAEGER pnpm next build test/integration/basic",
```

Comment:
@timneutkens Is `clean-trace-jaeger` still used? I changed this to use `pnpm next` but I realize now that the `test/integration/basic` dir doesn't exist

Reply:
Yeah I think it's leftover we can remove in a follow-up PR if we want